src/main/java/com/minecolonies/util/AchievementUtils.java

```diff
@@ -0,0 +1,29 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+import net.minecraft.stats.Achievement;
+
+import java.util.List;
+
+/**
+ *
+ */
+public final class AchievementUtils {
+
+    public static void syncAchievements(final Colony colony) {
+        final List<Permissions.Player> players = PermissionUtils.getPlayersWithAtleastRank(colony, Permissions.Rank.OFFICER);
+
+        for (Permissions.Player player : players) {
+            for (Achievement achievement: colony.getAchievements()) {
+                ServerUtils.getPlayerFromUUID(player.getID()).triggerAchievement(achievement);
+            }
+        }
+    }
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)